### DISCUSSION TITLE
the functionality of `nuxt/content`

### DISCUSSION THREAD

**Discussion by Sazzels** [nuxt/content#2479] [General]
I'm a bit confused about the functionality of `nuxt/content`. I find myself in a loop where I have to perform build steps every time I want to update the content. This seems counterintuitive, especially when my goal is to build an application that dynamically crawls content from a given folder (`content`) and allows seamless updates to a given target folder (say, the contents of `.output`). My ideal scenario is to avoid having to ship the entire codebase to clients each time a content change is made or to give them access. In general, file based CMS with `*.md` sounds very good for small businesses without having to host a backend CMS like `strapi`. 

I'm interested in developing a system that decouples content updates from code deployment. In this setup, changes to the content folder would be reflected directly in the output folder without the need for a full code build. I believe this approach could significantly streamline the process, ensuring that only relevant content updates are pushed to clients, rather than repeatedly deploying the entire code base.

Can you provide any guidance or suggestions on how to achieve such an architecture, or clarify how `nuxt/content` can be aligned with such a workflow?

___

In a nutshell, what i'm aiming for is a user-friendly system where non-coders can effortlessly update content without delving into the complexities of the underlying code. This setup typically involves creating a clear separation between the content management interface and the technical codebase.

Ideally, non-coders should interact with a user-friendly content management system (CMS) or interface tailored specifically for content updates. This interface should abstract away any intricate code details and provide a straightforward environment for editing, adding, or removing content. (this could in the simplest way beeing editing the `*.md` below the `content` folder)

By establishing this separation between the technical code and content management, empowering non-coders to maintain and update content autonomously. This approach enhances efficiency, allowing content updates without the risk of inadvertently altering or disrupting the codebase.

_P.S. I'm pretty new to the whole Nuxt ecosystem and modules, maybe my understanding of the workflow in this architecture isn't that deep at this point or I'm thinking way too simple._

**Comment by oripka** [comment]
https://nuxt.studio/

**Comment by Sazzels** [comment]
@oripka Not every one of my customers/projects has a repo at Github and can offer a public repo for this, although it looks very interesting and I am considering it for an upcoming project. A self-hosted `nuxt/studio` would be nice.

I want to get markdown from an API and then load it into the renderer. Can I do this standalone with `nuxt-modules/mdc`?
